### DISCUSSION TITLE
Plex LXC won't install: /dev/dri/renderD128 is not a device

### DISCUSSION THREAD

**Discussion by canuckcam** [tteck/Proxmox#3613] [Help Needed]
`pve-manager/8.2.4/faa83925c9641325 (running kernel: 6.8.12-1-pve)`

Hello, I had a Plex LXC, got a Tesla P4 to try to get hw transcoding working. I installed a wrong NVIDIA driver, uninstalled that via single user mode (Proxmox wouldn't boot), installed NVIDIA 535.161.05 following the PolloLoco instructions, verified working but when checking the Plex LXC, it refused to automatically login and after a minute Proxmox would hang. Proxmox has been stable not running the Plex container. I didn't make a back up (roast away) so wanted to make a new instance and transfer the settings over. However, the script now errors out on installation as follows:

```
Creating a Plex LXC using the above default settings
 ✓ Using local for Template Storage.
 ✓ Using local-zfs for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 119 was successfully created.
 //dev/dri/renderD128 is not a device

[ERROR] in line 624: exit code 0: while executing command pct start "$CTID"
```

but renderD128 does exist!?, noticed permissions so chmod 666 /dev/dri/renderD128
```
root@pve:~# ls -l /dev/dri
total 0
drwxr-xr-x 2 root root      60 Aug 29 01:17 by-path
crw-rw---- 1 root video 226, 0 Aug 29 01:17 card0
---------- 1 root root       0 Aug 29 01:18 renderD128

root@pve:/etc/pve/lxc# ls -l /dev/dri
total 0
drwxr-xr-x 2 root root      60 Aug 29 01:17 by-path
crw-rw---- 1 root video 226, 0 Aug 29 01:17 card0
-rw-rw-rw- 1 root root       0 Aug 29 01:18 renderD128
```

Run script again - this time the slash is a backslash.
```
 \/dev/dri/renderD128 is not a device
```

nvidia-smi returns ok.
```
+---------------------------------------------------------------------------------------+
| NVIDIA-SMI 535.161.05             Driver Version: 535.161.05   CUDA Version: N/A      |
|-----------------------------------------+----------------------+----------------------+
| GPU  Name                 Persistence-M | Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp   Perf          Pwr:Usage/Cap |         Memory-Usage | GPU-Util  Compute M. |
|                                         |                      |               MIG M. |
|=========================================+======================+======================|
|   0  Tesla P4                       On  | 00000000:15:00.0 Off |                    0 |
| N/A   48C    P8              11W /  75W |     31MiB /  7680MiB |      0%      Default |
|                                         |                      |                  N/A |
+-----------------------------------------+----------------------+----------------------+
                                                                                         
+---------------------------------------------------------------------------------------+
| Processes:                                                                            |
|  GPU   GI   CI        PID   Type   Process name                            GPU Memory |
|        ID   ID                                                             Usage      |
|=======================================================================================|
|  No running processes found                                                           |
+---------------------------------------------------------------------------------------+
```

If I run my Windows VM, nvidia-smi does register a vGPU instance
```
|    0   N/A  N/A      2092    C+G   vgpu                                       1872MiB |
```
mdevctl types returns ok
```
root@pve:~# mdevctl types
0000:15:00.0
  nvidia-157
    Available instances: 3
    Device API: vfio-pci
    Name: GRID P4-2B
```

Clearly I'm a bit of a n00b who can get into trouble but not know how to get out of it. Appreciate any help.

**Comment by wayne83m** [comment]
i have the exact same error

**Comment by fliberdygibits** [comment]
I just had this same issue a few days ago tho I suspect my answer won't be your answer.  I was getting this on a lenovo m720q with the little add-on DP card.  I plugged a cable in to get into the bios and when I restarted it was reporting that same error....  I FINALLY removed the cable on a whim and it went away.  SO I did some testing with other cables and the other ports on the system.  Near as I can tell my otherwise perfectly good DP to DP cable when plugged in to that add in card somehow kills the integrated GPU.

**Comment by canuckcam** [comment]
I do have an NVS 310 card for BIOS/Proxmox, which is old enough that the drivers warn it does not address the NVS 310 so I assume(!?) it just ignores it. 

Now, from what I can understand is vGPU to VM is ok because Proxmox doesn't take control of the GPU (the blacklist) but since LXCs are directly related to the Proxmox kernel, it is one or the other (sharing 1 physical GPU amongst all LXCs or all VMs). But I can't get the LXCs to recognize any GPU.
